### ISSUE TITLE
Perhaps, we could make it so that every time you KO a mon, there is a chance for the same species to spawn, and that chance grows each time. Evolutions have a chance to spawn instead, however, that will break your chain if you KO/catch it.

### ISSUE THREAD

**Issue by tmetcalfe89** [timinc-cobble/cobblemon-unchained-1.4-fabric#13]

**Comment by RiftSpatial** [comment]
Increase in the number of possible spawns after each threshold, perhaps we can set a limit of how many of that specific species can spawn, so it won't attempt to spawn three when there are already, say, 5 Wooloo (it's what I've been thinking of this whole time because of Sword and Shield). This keeps it from spawning too much, which can be seen as OP, while also, rewarding you even more for a higher streak.

**Comment by tmetcalfe89** [comment]
I don't think this would be in scope for this mod. It'd take something like an additional spawn to "feel good" imo. I do have the idea of a shaking grass mod.
